CoThings/Models.swift

```diff
@@ -11,6 +11,9 @@ import Foundation
 let ServerHostNameKey = "serverHostname"
 let PassOnboardingKey = "passOnboarding"
 let LastEnteredRoomIdKey = "lastEnteredRoomId"
+let NotifyOnEnterKey = "notifyOnEnter"
```

Comment:
What is the situation where the user wants to get notifications for enter or exit without specifying a special room. I wouldn't like to get all the irrelevant notifications from other rooms while only waiting for the bathroom to be available

Reply:
nah this is not that feature, yet. for that we need server side implementation.. this is only for let use know when they entered a room or exited from a room so they don't have to manually update it. 

It would be also good to let them know that they have already in the room in the UI (Home screen), or disabled -/+ buttons. But they might want to adjust the number, not everyone going to use the app..

Reply:
I see, it is okay as far as it is opt in.